### PR TITLE
Update milv image

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Update `milv` to `0.0.7-alpha` version: Add `black-list-internal` and `black-list-external` flags for not checking relative and external links in provided path.

**Related issue(s)**
See https://github.com/kyma-project/website/issues/103

### CODE REVIEW

File: prow/scripts/governance.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPT_DIR}/library.sh"
 
 readonly ARGS=("$@")
-readonly MILV_IMAGE="magicmatatjahu/milv:0.0.6"
+readonly MILV_IMAGE="magicmatatjahu/milv:0.0.7-alpha"
```
Comment (michal-hudy) [inline prow/scripts/governance.sh]: How stable is `alpha`? Did you test it against all of our repositories where governance is enabled?

Comment (magicmatatjahu) [reply prow/scripts/governance.sh]: `alpha` has the ability to ignore checking internal links in given files/folders and I tested it in some example cases, but before release `0.0.7` version I want to add a few features that do not affect the task.

And yes, I tested new version in all of our repositories where governance is enabled 😸

Comment (michal-hudy) [reply prow/scripts/governance.sh]: Perfect 💯 Approved 👏

Comment (pkosiec) [reply prow/scripts/governance.sh]: BTW @magicmatatjahu why did you name the version `0.0.7-alpha`, not `0.0.7`, if everything is well-tested? 😄

Comment (magicmatatjahu) [reply prow/scripts/governance.sh]: @pkosiec `...but before release 0.0.7 version I want to add a few features that do not affect the task.` 😄

**Pull request merged**